File: python/22859.py
import re

for title, p in re.findall(r"<div title=\"([^\"]+)\">(.+?)</div>", input()):
    print(f"title : {title}")

    for inside_p in re.findall(r"<p>(.+?)</p>", p):
        paragraph = re.sub(r"<[^>]+>", "", inside_p)
        paragraph = re.sub(r"\s+", " ", paragraph).strip()

        print(paragraph)

File: javascript/22859.js
const input = require("fs").readFileSync("/dev/stdin").toString().trim();

for (const [_, title, p] of input.matchAll(/<div title="([^"]+)">(.+?)<\/div>/g)) {
  console.log(`title : ${title}`);

  for (const [_, inside_p] of p.matchAll(/<p>(.+?)<\/p>/g)) {
    const paragraph = inside_p
      .replaceAll(/<[^>]+>/g, "")
      .replaceAll(/\s+/g, " ")
      .trim();

    console.log(paragraph);
  }
}
